packages/data-stores/src/auth/index.ts

```diff
@@ -1,20 +1,15 @@
-/**
- * External dependencies
- */
 import { registerStore } from '@wordpress/data';
 import { controls as dataControls } from '@wordpress/data-controls';
 import { requestAllBlogsAccess } from 'wpcom-proxy-request';
 
```

Comment:
Is leaving the empty line between the 2 blocks intentional? I think it would make sense to remove it at this point when we no longer have distinct blocks of imports. Totally minor, of course.

Reply:
I asked myself the same question yesterday when reviewing #54452 and it seems like an empty line is intentional between groups of imports (see [here](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#newlines-between-ignorealwaysalways-and-inside-groupsnever)).

Reply:
Makes sense to me! Thanks for confirming @flootr.

Reply:
We can of course change the `newlines-between` config, currently set to `always`. Personally I think `always` looks "cleaner", but I don't have strong opinions either way. What do you think?

Reply:
No strong opinions either. If you ask me, I'd prefer no line because now that we have no docblocks, it's even less obvious what that line is for, and it's also less useful.

Reply:
Agree, unless you know `import/order` has the concept of 'groups, the blank line is probably confusing. I'll remove it.